presto-oracle/src/test/java/io/prestosql/plugin/oracle/OracleServer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.oracle;
+
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.OracleContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+
+public class OracleServer
+        extends OracleContainer
+        implements Closeable
+{
+    public static final String TEST_SCHEMA = "tpch";
+    public static final String TEST_USER = "tpch";
+    public static final String TEST_PASS = "oracle";
+
+    public OracleServer()
+    {
+        super("wnameless/oracle-xe-11g-r2");
+
+        // this is added to allow more processes on database, otherwise the tests end up giving a ORA-12519
+        // to fix this we have to execute this command "alter system set processes=300 scope=spfile"
+        // but this command needs a database restart and if we restart the docker the configuration is lost.
+        this.addFileSystemBind("src/test/resources/spfileXE.ora",
+                "/u01/app/oracle/product/11.2.0/xe/dbs/spfileXE.ora",
+                BindMode.READ_WRITE);
```

Comment:
bind READ_ONLY, since this file is not supposed to be changed by the container

Reply:
correct..